docs/en/latest/FAQ.md

```diff
@@ -371,3 +371,36 @@ You may have to install OPENSSL in your system and/or pass OPENSSL_DIR or OPENSS
 Example: luarocks install luasec OPENSSL_DIR=/usr/local
 make: *** [deps] Error 1
 ```
+
+## How to access APISIX Dashboard through APISIX proxy
+
+1. Keep the APISIX proxy port and Admin API port different(or disable Admin API). For example, do the following configuration in `conf/config.yaml`.
+
+The Admin API use a separate port 9180:
+
+```yaml
+apisix:
+  port_admin: 9180            # use a separate port
+```
+
+2. Add proxy route of APISIX Dashboard:
+
+Note: The APISIX Dashboard service here is listening on `127.0.0.1:9000`.
+
+```shell
+curl -i http://127.0.0.1:9180/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "uris":[ "/*" ],
```

Comment:
Does this mean all uris will match this route? will this affect other routes?

Reply:
For non-existent uri, it will match "/*". If the uri has been configured, it will not be affected. For example, if the uri of a certain route is `/hello*`, then all requests prefixed with `/hello` will match `/hello*` instead of `/*`. Therefore, this will not affect other existing routes.